lib/highline.rb

```diff
@@ -668,7 +439,7 @@ def page_at=( setting )
   # Outputs indentation with current settings
   #
   def indentation
-    return ' '*@indent_size*@indent_level
+    ' '*@indent_size*@indent_level
```

Comment:
Yes! Sure.

I think, after I approach to the end of this refactoring I'll try to use Rubotto to catch all of these stylistic problems.